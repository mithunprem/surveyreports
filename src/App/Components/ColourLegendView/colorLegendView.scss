.color-legend {
  width: 300px;
}
