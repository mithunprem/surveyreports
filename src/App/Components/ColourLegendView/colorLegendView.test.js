import React from 'react';
import { create } from 'react-test-renderer';
import ColourLegendView from '../../Components/ColourLegendView';

describe("ColourLegendView", () => {
  test("should render without throwing an error", () => {
    const colorLegendView = create(<ColourLegendView />).toJSON();
    expect(colorLegendView).toMatchSnapshot();
  });
});
