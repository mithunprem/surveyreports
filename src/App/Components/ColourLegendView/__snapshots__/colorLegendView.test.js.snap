// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ColourLegendView should render without throwing an error 1`] = `
<div
  className="ml-3 mb-4 color-legend"
>
  <span
    className="small"
  >
    Color legend :
  </span>
  <div
    className="progress"
  >
    <div
      aria-valuemax={100}
      aria-valuemin="0"
      aria-valuenow="33.3"
      className="progress-bar bg-danger"
      role="progressbar"
      style={
        Object {
          "width": "33.3%",
        }
      }
    >
      Rating &lt; 3.5
    </div>
    <div
      aria-valuemax={100}
      aria-valuemin="0"
      aria-valuenow="33.3"
      className="progress-bar bg-warning"
      role="progressbar"
      style={
        Object {
          "width": "33.3%",
        }
      }
    >
      3.5 &lt; Rating &lt; 4
    </div>
    <div
      aria-valuemax={100}
      aria-valuemin="0"
      aria-valuenow="33.3"
      className="progress-bar bg-success"
      role="progressbar"
      style={
        Object {
          "width": "33.3%",
        }
      }
    >
      Rating &gt; 4
    </div>
  </div>
</div>
`;
